static/js/marker.js

```diff
@@ -43,7 +43,7 @@ var MarkerView = Backbone.View.extend({
 
         if (this.model.get("provider_code") == PROVIDER_CODE_UNITED_HATZALA) {
             this.marker.setIcon(this.getIcon());
-            this.marker.setTitle(this.getTitle('united'));
+            this.marker.setTitle(this.model.get("title"));
```

Comment:
It's a new `switch` @LironRS has constructed for the markers.
Specifically the united markers are getting their title and description straight from the DB.
I should have removed this `case`.
Do you see any reason to use the switch for united too instead of the data from the marker model?